tests/src/main/scalajs/sttp/tapir/tests.scala

```diff
@@ -0,0 +1,8 @@
+package sttp.tapir
+
+import org.scalajs.dom.File
+
+package object fileTest {
```

Comment:
maybe we could do it same as in other cases, that is create a `TestsExtensions` trait, and the package object can extend that